bees/prometheusbee/prometheusbeefactory.go

```diff
@@ -0,0 +1,284 @@
+/*
+ *    Copyright (C) 2014-2017 Christian Muehlhaeuser
```

Comment:
Forgot to add your copyright here! :smile:

Reply:
Whoops!